Bottom: 843795995432c0ddfedd5fb6583ce99c469402db
Top:    28f650231918ea054dd5bd0dda9bf752b04a09b7
Author: Peter Clifton <pcjc2@cam.ac.uk>
Date:   2009-10-14 14:43:10 +0100

Refresh of use-trackball-to-allow-rotatio

---

diff --git a/src/hid/gtk/gui-output-events.c b/src/hid/gtk/gui-output-events.c
index 520dd99..a8404e9 100644
--- a/src/hid/gtk/gui-output-events.c
+++ b/src/hid/gtk/gui-output-events.c
@@ -876,7 +876,7 @@ ghid_port_drawing_area_expose_event_cb (GtkWidget * widget,
   glMultMatrixf (view_matrix);
   glTranslatef (-widget->allocation.width / 2., -widget->allocation.height / 2., 0);
 
-//  glEnable (GL_STENCIL_TEST);
+  glEnable (GL_STENCIL_TEST);
   glClearColor (gport->offlimits_color.red / 65535.,
                 gport->offlimits_color.green / 65535.,
                 gport->offlimits_color.blue / 65535.,
